app/utils/pusher.js

```diff
@@ -156,22 +171,12 @@ TravisPusher.prototype.ignoreMessage = function (message) {
   return existingSubscription || noSubscription;
 };
 
-Pusher.SockJSTransport.isSupported = function () {
-  if (ENV.pusher.host !== 'ws.pusherapp.com') {
-    return false;
-  }
-};
-
-Pusher.channel_auth_transport = 'travis_ajax';
```

Comment:
Unfortunately this needs to stay as is for now. API implements pusher auth differently than Pusher, because in the past when we were using a channel per repo approach we needed to auth in bulk. We can change it in the API, but until we do, we need a custom authorizer.

Reply:
I think this is still working, they just changed the interface, so I moved it into the `authorize` method.

Reply:
👍